nservicebus/azure-storage-persistence/configuration.md

```diff
@@ -54,4 +56,4 @@ For Timeouts:
 
 snippet:AzurePersistenceTimeoutsCustomization
 
-NOTE: Subscriptions and Timeouts persistence configuration only has an effect when used with Azure Storage Queues transport from NServiceBus Azure Version 6 and later.
\ No newline at end of file
+NOTE: Subscriptions and Timeouts persistence configuration only has an effect when used with Azure Storage Queues transport from NServiceBus.Azure Version 6.x and NServiceBus.Persistence.AzureStorage Version 1.x and above.
```

Comment:
Ah I see now that the this is 6+ and 1+ so if you were in 5- then those specific configurations won't work. Maybe it's easier to phrase it that way?

> Subscription and Timeout persistence configuration has no effect in `NServiceBus.Azure` Version 5 and below.